p4sdk/images/entrypoint.sh

```diff
@@ -43,6 +43,7 @@ mkdir -p $CONF_DIR
 envsubst < $CONF_FILE.template > $CONF_DIR/$CONF_FILE
 
 touch /opt/p4/rh_mvp/tofino.bin
+sleep 5
```

Comment:
This sleep is also not needed

Reply:
remove all sleep as it turned out to be the issue with hugepages.